math_keyboard/lib/src/widgets/keyboard_button.dart

```diff
@@ -51,6 +51,11 @@ class _KeyboardButtonState extends State<KeyboardButton>
     super.dispose();
   }
 
+  void _handleHover(bool entered) {
+    _animationController.value = entered ? 0.5 : 0;
+    widget.onHold?.call();
```

Comment:
@tomkuschka By calling `onHold` here you will enter symbols on hover, don't you?